### PR TITLE
Add form

### PR DESCRIPTION
**Pull request by SSEKPIUS**
Modified index.html , style.css and README file to add contact form and button interactions

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -285,6 +285,55 @@ <h2>
...
+            </li>
+            <li>
+                <textarea class="contact-3" maxlength="500" placeholder="Enter text here" required></textarea>
+            </li>
+        </ul>
+        <div class="contact-button">
+            <button type="submit" class="button-submit">
+                Get in Touch
+            </button>
+        </div>
```
Comment (Abenezer-Tilahun) [inline index.html:319]: - [ ] As stated in the Project requirement I recommend you to Implement the button interactions such as hover, active (when it's pressed), disabled and enable (default state) are styled as shown in the Figma template.
  
Capture

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @SSEKPIUS    , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct GitHub Flow.
✔️ Professional   README
✔️ Good commit messages

almost

## Required Changes ♻️
_Check the comments below the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -389,4 +389,133 @@ span {
...
+  width: 327px;
+  border-radius: 4px;
+  border: 1px solid #d0d9d4;
+}
+
+.contact-2 {
+  height: 48px;
+  width: 327px;
+  border-radius: 4px;
+  border: 1px solid #d0d9d4;
```
Comment (christianbiring1) [inline style.css:466]: [ OPTIONAL] 
- [ ] Kindly make sure your page is not overlapping on the right. as you can see
image
Make sure you use a relative unit like `%`or `vw`.  As you can notice here your width is to `327px` whereas the minum mobile width is `320px`

File: index.html (hunk 1 above)
Comment (christianbiring1) [reply index.html:319]: - [x] As said by the previous reviewer, Kindly consider implementing the buttons interaction. `hover`, `disable`, and `active` for it helps the user to see the action they are about to make on your page.

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @TayyabKM , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
# STATUS: REQUEST CHANGES :tada: 
soclose


 ## Highlights
- [x] All linters checks are green :heavy_check_mark: 
- [x] Descriptive PR `title`

 ## Required Changes ♻️
 - [x] Kindly make sure you use the correct form id so that user can have the ability to submit the form successfully. 
image


`hint: YOU CAN CHECK` [HERE](https://www.youtube.com/results?search_query=formspree) `HOW TO USE FROMSPREE`
_Check the comments under the review._

## Optional suggestions
#### I highly recommend you implement them for they will help take your project to the next level
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @SSEKPIUS 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

 - No linter errors. ✔️ 
 - Form working perfectly. ✔️ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**